src/App.tsx

```diff
@@ -1,155 +1,73 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+
+import { TodoStatus } from './enums/TodoStatus';
+import { ErrorMessage } from './enums/ErrorMessage';
+
+import { getTodos } from './api/todos';
+
+import { filterTodos } from './utils/filterTodos';
+
+const USER_ID = 6522;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<TodoType[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState(TodoStatus.All);
+  const [errorMessage, setErrorMessage] = useState(ErrorMessage.NONE);
+  const [isErrorShown, setIsErrorShown] = useState(false);
```

Comment:
there is no need to use additional state value `isErrorShown`. you can check by `errorMessage` 
```suggestion
  const [errorMessage, setErrorMessage] = useState(ErrorMessage.NONE);
```